pkg/sql/sessiondatapb/local_only_session_data.proto

```diff
@@ -246,6 +246,10 @@ message LocalOnlySessionData {
   // (with no column name specifiers) to expect and ignore not visible column
   // fields.
   bool expect_and_ignore_not_visible_columns_in_copy = 67;
+  // ShowPrimaryKeyConstraintOnNotVisibleColumns controls whether SHOW
+  // CONSTRAINTS and pg_catalog.pg_constraint will include primary key
+  // constraints that only include hidden columns.
+  bool show_primary_key_constraint_on_not_visible_columns = 68;
```

Comment:
While trying to backport the setting before this one, I noticed that in master this field has field number 69, not 68. Could that cause a problem in mixed-version clusters? Maybe it's ok because this is local-only? I don't think we've released this yet, so I'm going to change the field number to match master in https://github.com/cockroachdb/cockroach/pull/79930

Reply:
yeah if it's local-only it shouldn't be a problem, since the setting only matters on the gateway node. but making it match anyway sounds good.